precommit/src/main/shell/plugins.d/junit.sh

```diff
@@ -25,16 +25,21 @@ JUNIT_FAILED_TESTS=""
 JUNIT_TEST_OUTPUT_DIR="."
 JUNIT_TEST_PREFIX="org.apache."
 
+JUNIT_REPORT_STYLE="line"
```

Comment:
Are we sure we want change the default report style on people? Needs a big release note...

Reply:
Looking at a lot of different junit output implementations over the past week, our current form is very much outside the norm. There will definitely be a release note, but long term I feel strongly that this approach is generally better.

Reply:
(One thing to keep in mind is that our current approach basically hides how many _actual_ errors there are; instead it is a count of how many tests failed!)